collections/St_Johns_College/SW_V1_St_Johns_College_MS_006.xml

```diff
@@ -0,0 +1,233 @@
+               <email>specialcollections.enquiries@bodleian.ox.ac.uk</email>
+            </distributor>
+            <idno type="msID">St_Johns_College_MS_6</idno>
+            <idno type="collection">St_Johns_College</idno>
+            <idno type="catalogue">Western</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:lang="en" xml:id="St_Johns_College_MS_6">
+               <msIdentifier>
+                  <country>United Kingdom</country>
+                  <region type="county">Oxfordshire</region>
+                  <settlement key="7011931">Oxford</settlement>
+                  <institution>St John's College</institution>
+                  <repository>St John's College</repository>
+                  <idno type="shelfmark">St John's College MS. 6</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="enm">English. LALME places the language in <placeName
+                        key="place_7008143">Herefordshire</placeName> (grid 345 269), LP 7510
+                     (3:174). <placeName key="">Leinthall</placeName> (see ‘Provenance’) is six miles
+                     south-west of <placeName key="place_7012100">Ludlow</placeName>. </textLang>
+                  <msItem n="1" xml:id="St_Johns_College_MS_6-item1">
+                     <locus from="1ra" to="134ra">Fols. 1<hi rend="superscript">ra</hi>–134<hi
+                           rend="superscript">ra</hi></locus>
+                     <rubric>Here bygynneþ þe prolog of the seege of Troye in englischs translated
+                        after the latyn compiled by Gwydo de Columpnis not word by word</rubric>
+                     <incipit>O myghty Mars that with thy sturne light | In armes hast the
+                        powwer</incipit>
+                     <finalRubric><locus from="133va" to="133va">[fol. 133<hi rend="superscript"
+                              >va</hi>]</locus> Thus endeth þe fyfþe booke of þe seege of troye
+                        vnder þe correccioun of euery prudent Reeder Quod LIDGATE Monke of þe
+                        Burye</finalRubric>
+                     <explicit>withdrawe and goo abak | Requerynge hem alle þat is mys to
+                        amende</explicit>
+                     <colophon>Quod <persName key="person_4617" role="scr">Ion
+                              Sch<supplied>row</supplied>sbury</persName></colophon>
+                     <author key="person_22158282">John Lydgate</author>
+                     <title key="work_2732">The Troy Book</title>
+                     <note><bibl>(IMEV 2516)</bibl>, <bibl>ed. Thomas Bergen, EETS ES 97, 103, 106
+                           (1906–10)</bibl> , with the envoy and ‘Verba translatoris’ (the actual
+                        explicit). Bergen describes our MS at <bibl>EETS ES 126 (1935) 36–8.</bibl>
+                        <locus from="134rb" to="134v">Fol. 134<hi rend="superscript"
```

Comment:
I would divide the note element into two at this point (i.e. after `36-8.</bibl>`), for improved legibility.